src/app/zap-templates/partials/idl/cluster_attribute.zapt

```diff
@@ -0,0 +1,26 @@
+  {{! ensure indent}}{{#unless isWritable~}}
+  readonly {{!marker to place a space even with whitespace removal~}}
+{{~/unless~}}
+{{~!TODO: write only attributes should also be supported~}}
+{{~!TODO: non-reportable (nosubscribe) attributes should also be supported ~}}
+{{~!Removes spaces~}} attribute {{!ensure space}}
+  {{~#chip_access_elements entity="attribute"~}}
+    {{~#first~}}access({{~/first~}}
+    {{~#not_first~}}, {{/not_first~}}
+      {{operation}}: {{role}}
+    {{~#last}}) {{/last~}}
+  {{~/chip_access_elements~}}
+  {{~#if isNullable~}}
+    nullable {{!marker to place a space even with whitespace removal~}}
+  {{~/if~}} {{#if entryType}}{{entryType}}{{else}}{{type}}{{/if}}
```

Comment:
You can use `{{#if isArray}}` if under zcl_attributes_server block helper

Reply:
It did not work for me. isArray was always false (notice that isArray was already used in this method)

Reply:
It should work with today's ZAP release.